diff --git a/.vs/Bullet_Fire/v15/.suo b/.vs/Bullet_Fire/v15/.suo
index b82077c..8184b98 100644
Binary files a/.vs/Bullet_Fire/v15/.suo and b/.vs/Bullet_Fire/v15/.suo differ
diff --git a/Library/CurrentLayout-default.dwlt b/Library/CurrentLayout-default.dwlt
index f3f2489..5d52842 100644
--- a/Library/CurrentLayout-default.dwlt
+++ b/Library/CurrentLayout-default.dwlt
@@ -14,14 +14,14 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   m_PixelRect:
     serializedVersion: 2
-    x: -1920
-    y: 43
-    width: 1920
-    height: 997
+    x: 1920
+    y: -312
+    width: 2560
+    height: 1357
   m_ShowMode: 4
   m_Title: Project
   m_RootView: {fileID: 2}
-  m_MinSize: {x: 950, y: 444}
+  m_MinSize: {x: 950, y: 300}
   m_MaxSize: {x: 10000, y: 10000}
 --- !u!114 &2
 MonoBehaviour:
@@ -43,8 +43,8 @@ MonoBehaviour:
     serializedVersion: 2
     x: 0
     y: 0
-    width: 1920
-    height: 997
+    width: 2560
+    height: 1357
   m_MinSize: {x: 950, y: 300}
   m_MaxSize: {x: 10000, y: 10000}
 --- !u!114 &3
@@ -64,7 +64,7 @@ MonoBehaviour:
     serializedVersion: 2
     x: 0
     y: 0
-    width: 1920
+    width: 2560
     height: 30
   m_MinSize: {x: 0, y: 0}
   m_MaxSize: {x: 0, y: 0}
@@ -85,8 +85,8 @@ MonoBehaviour:
   m_Position:
     serializedVersion: 2
     x: 0
-    y: 977
-    width: 1920
+    y: 1337
+    width: 2560
     height: 20
   m_MinSize: {x: 0, y: 0}
   m_MaxSize: {x: 0, y: 0}
@@ -109,12 +109,12 @@ MonoBehaviour:
     serializedVersion: 2
     x: 0
     y: 30
-    width: 1920
-    height: 947
-  m_MinSize: {x: 1115, y: 394}
-  m_MaxSize: {x: 26010, y: 14044}
+    width: 2560
+    height: 1307
+  m_MinSize: {x: 1121, y: 394}
+  m_MaxSize: {x: 26016, y: 14044}
   vertical: 0
-  controlID: 38
+  controlID: 152
 --- !u!114 &6
 MonoBehaviour:
   m_ObjectHideFlags: 52
@@ -132,8 +132,8 @@ MonoBehaviour:
     serializedVersion: 2
     x: 0
     y: 0
-    width: 738
-    height: 947
+    width: 984
+    height: 1307
   m_MinSize: {x: 202, y: 222}
   m_MaxSize: {x: 4002, y: 4022}
   m_ActualView: {fileID: 16}
@@ -158,14 +158,14 @@ MonoBehaviour:
   - {fileID: 13}
   m_Position:
     serializedVersion: 2
-    x: 738
+    x: 984
     y: 0
-    width: 1182
-    height: 947
-  m_MinSize: {x: 913, y: 394}
-  m_MaxSize: {x: 22008, y: 14044}
+    width: 1576
+    height: 1307
+  m_MinSize: {x: 919, y: 394}
+  m_MaxSize: {x: 22014, y: 14044}
   vertical: 1
-  controlID: 39
+  controlID: 153
 --- !u!114 &8
 MonoBehaviour:
   m_ObjectHideFlags: 52
@@ -187,12 +187,12 @@ MonoBehaviour:
     serializedVersion: 2
     x: 0
     y: 0
-    width: 1182
-    height: 679
-  m_MinSize: {x: 913, y: 272}
-  m_MaxSize: {x: 22008, y: 10022}
+    width: 1576
+    height: 937
+  m_MinSize: {x: 919, y: 272}
+  m_MaxSize: {x: 22014, y: 10022}
   vertical: 0
-  controlID: 40
+  controlID: 154
 --- !u!114 &9
 MonoBehaviour:
   m_ObjectHideFlags: 52
@@ -210,8 +210,8 @@ MonoBehaviour:
     serializedVersion: 2
     x: 0
     y: 0
-    width: 461
-    height: 679
+    width: 614
+    height: 937
   m_MinSize: {x: 204, y: 222}
   m_MaxSize: {x: 4004, y: 4022}
   m_ActualView: {fileID: 17}
@@ -236,12 +236,12 @@ MonoBehaviour:
   m_Children: []
   m_Position:
     serializedVersion: 2
-    x: 461
+    x: 614
     y: 0
-    width: 206
-    height: 679
-  m_MinSize: {x: 200, y: 200}
-  m_MaxSize: {x: 4000, y: 4000}
+    width: 275
+    height: 937
+  m_MinSize: {x: 204, y: 222}
+  m_MaxSize: {x: 4004, y: 4022}
   m_ActualView: {fileID: 19}
   m_Panes:
   - {fileID: 19}
@@ -262,10 +262,10 @@ MonoBehaviour:
   m_Children: []
   m_Position:
     serializedVersion: 2
-    x: 667
+    x: 889
     y: 0
-    width: 236
-    height: 679
+    width: 315
+    height: 937
   m_MinSize: {x: 234, y: 272}
   m_MaxSize: {x: 10004, y: 10022}
   m_ActualView: {fileID: 20}
@@ -288,12 +288,12 @@ MonoBehaviour:
   m_Children: []
   m_Position:
     serializedVersion: 2
-    x: 903
+    x: 1204
     y: 0
-    width: 279
-    height: 679
-  m_MinSize: {x: 275, y: 50}
-  m_MaxSize: {x: 4000, y: 4000}
+    width: 372
+    height: 937
+  m_MinSize: {x: 277, y: 72}
+  m_MaxSize: {x: 4002, y: 4022}
   m_ActualView: {fileID: 21}
   m_Panes:
   - {fileID: 21}
@@ -317,9 +317,9 @@ MonoBehaviour:
   m_Position:
     serializedVersion: 2
     x: 0
-    y: 679
-    width: 1182
-    height: 268
+    y: 937
+    width: 1576
+    height: 370
   m_MinSize: {x: 102, y: 122}
   m_MaxSize: {x: 4002, y: 4022}
   m_ActualView: {fileID: 15}
@@ -374,10 +374,10 @@ MonoBehaviour:
     m_Tooltip: 
   m_Pos:
     serializedVersion: 2
-    x: -1182
-    y: 752
-    width: 1180
-    height: 249
+    x: 2904
+    y: 655
+    width: 1574
+    height: 351
   m_ViewDataDictionary: {fileID: 0}
 --- !u!114 &16
 MonoBehaviour:
@@ -400,10 +400,10 @@ MonoBehaviour:
     m_Tooltip: 
   m_Pos:
     serializedVersion: 2
-    x: -1920
-    y: 73
-    width: 738
-    height: 928
+    x: 1920
+    y: -282
+    width: 984
+    height: 1288
   m_ViewDataDictionary: {fileID: 0}
   m_VSyncEnabled: 0
   m_MaximizeOnPlay: 0
@@ -437,10 +437,10 @@ MonoBehaviour:
       serializedVersion: 2
       x: 0
       y: 17
-      width: 738
-      height: 911
-    m_Scale: {x: 0.24932432, y: 0.2493243}
-    m_Translation: {x: 369, y: 455.5}
+      width: 984
+      height: 1271
+    m_Scale: {x: 0.33243242, y: 0.33243242}
+    m_Translation: {x: 491.99997, y: 635.5}
     m_MarginLeft: 0
     m_MarginRight: 0
     m_MarginTop: 0
@@ -448,14 +448,14 @@ MonoBehaviour:
     m_LastShownAreaInsideMargins:
       serializedVersion: 2
       x: -1480
-      y: -1826.9377
+      y: -1911.6667
       width: 2960
-      height: 3653.8755
+      height: 3823.3335
     m_MinimalGUI: 1
-  m_defaultScale: 0.24932432
+  m_defaultScale: 0.33243242
   m_TargetTexture: {fileID: 0}
   m_CurrentColorSpace: 0
-  m_LastWindowPixelSize: {x: 738, y: 928}
+  m_LastWindowPixelSize: {x: 984, y: 1288}
   m_ClearInEditMode: 1
   m_NoCameraWarning: 1
   m_LowResolutionForAspectRatios: 01000001000000000000
@@ -481,10 +481,10 @@ MonoBehaviour:
     m_Tooltip: 
   m_Pos:
     serializedVersion: 2
-    x: -1182
-    y: 73
-    width: 459
-    height: 660
+    x: 2904
+    y: -282
+    width: 612
+    height: 918
   m_ViewDataDictionary: {fileID: 0}
   m_ShowContextualTools: 0
   m_WindowGUID: 64e9e1ce69284bf4e94f6f0f9c5cb1e7
@@ -644,17 +644,17 @@ MonoBehaviour:
     m_Tooltip: 
   m_Pos:
     serializedVersion: 2
-    x: -721
-    y: 73
-    width: 204
-    height: 660
+    x: 3518
+    y: -282
+    width: 273
+    height: 918
   m_ViewDataDictionary: {fileID: 0}
   m_SceneHierarchy:
     m_TreeViewState:
       scrollPos: {x: 0, y: 0}
       m_SelectedIDs: 
       m_LastClickedID: 0
-      m_ExpandedIDs: 3a9dfdff749dfdff209efdff289efdffaea5fdffd6a5fdfffea5fdff26a6fdff4ea6fdff76a6fdffe4a6fdff10a7fdff9ab6fdff0ab7fdffbe48ffff689affff22faffff76fbfffff4ffffff20320000783200005c360000b4360000b0370000fc370000
+      m_ExpandedIDs: 5afbffff
       m_RenameOverlay:
         m_UserAcceptedRename: 0
         m_Name: 
@@ -699,10 +699,10 @@ MonoBehaviour:
     m_Tooltip: 
   m_Pos:
     serializedVersion: 2
-    x: -515
-    y: 73
-    width: 234
-    height: 660
+    x: 3793
+    y: -282
+    width: 313
+    height: 918
   m_ViewDataDictionary: {fileID: 0}
   m_SearchFilter:
     m_NameFilter: 
@@ -729,7 +729,7 @@ MonoBehaviour:
     scrollPos: {x: 0, y: 0}
     m_SelectedIDs: 0e410000
     m_LastClickedID: 16654
-    m_ExpandedIDs: 00000000e22c0000082d00000a2d00000c2d0000242d0000622d0000da2d0000722e0000922e0000a02e0000a22f0000b22f0000f2310000
+    m_ExpandedIDs: 00000000d62c0000042d0000062d0000082d0000262d0000fe2d0000402e0000ae2e0000d02e0000e02e0000ec2f0000083000001a30000028300000b2320000
     m_RenameOverlay:
       m_UserAcceptedRename: 0
       m_Name: 
@@ -754,10 +754,10 @@ MonoBehaviour:
       m_Icon: {fileID: 0}
       m_ResourceFile: 
   m_AssetTreeState:
-    scrollPos: {x: 0, y: 737}
+    scrollPos: {x: 0, y: 479}
     m_SelectedIDs: 
     m_LastClickedID: 0
-    m_ExpandedIDs: ffffffff00000000e22c0000082d00000a2d00000c2d0000242d0000da2d0000722e0000922e0000a02e0000a22f0000b22f0000f2310000e6470000144800001a480000
+    m_ExpandedIDs: ffffffff00000000d62c0000042d0000062d0000082d0000262d0000fe2d0000402e0000ae2e0000d02e0000e02e0000ec2f0000083000001a30000028300000b2320000
     m_RenameOverlay:
       m_UserAcceptedRename: 0
       m_Name: 
@@ -834,10 +834,10 @@ MonoBehaviour:
     m_Tooltip: 
   m_Pos:
     serializedVersion: 2
-    x: -279
-    y: 73
-    width: 277
-    height: 660
+    x: 4108
+    y: -282
+    width: 370
+    height: 918
   m_ViewDataDictionary: {fileID: 0}
   m_OpenAddComponentMenu: 0
   m_ObjectsLockedBeforeSerialization: []
diff --git a/Library/SceneVisibilityState.asset b/Library/SceneVisibilityState.asset
index 60a169b..66e539a 100644
Binary files a/Library/SceneVisibilityState.asset and b/Library/SceneVisibilityState.asset differ
diff --git a/Library/ScriptAssemblies/Assembly-CSharp-Editor.dll b/Library/ScriptAssemblies/Assembly-CSharp-Editor.dll
deleted file mode 100644
index 3b4ebef..0000000
Binary files a/Library/ScriptAssemblies/Assembly-CSharp-Editor.dll and /dev/null differ
diff --git a/Library/ScriptAssemblies/Assembly-CSharp-Editor.pdb b/Library/ScriptAssemblies/Assembly-CSharp-Editor.pdb
deleted file mode 100644
index e5fa909..0000000
Binary files a/Library/ScriptAssemblies/Assembly-CSharp-Editor.pdb and /dev/null differ
diff --git a/Library/ScriptAssemblies/Assembly-CSharp.dll b/Library/ScriptAssemblies/Assembly-CSharp.dll
deleted file mode 100644
index 7b03df4..0000000
Binary files a/Library/ScriptAssemblies/Assembly-CSharp.dll and /dev/null differ
diff --git a/Library/ScriptAssemblies/Assembly-CSharp.pdb b/Library/ScriptAssemblies/Assembly-CSharp.pdb
deleted file mode 100644
index 9b98642..0000000
Binary files a/Library/ScriptAssemblies/Assembly-CSharp.pdb and /dev/null differ
diff --git a/Library/ScriptAssemblies/Unity.CollabProxy.Editor.dll b/Library/ScriptAssemblies/Unity.CollabProxy.Editor.dll
index 3cc73fa..07c1b23 100644
Binary files a/Library/ScriptAssemblies/Unity.CollabProxy.Editor.dll and b/Library/ScriptAssemblies/Unity.CollabProxy.Editor.dll differ
diff --git a/Library/ScriptAssemblies/Unity.Rider.Editor.dll b/Library/ScriptAssemblies/Unity.Rider.Editor.dll
index 63d5176..e95b546 100644
Binary files a/Library/ScriptAssemblies/Unity.Rider.Editor.dll and b/Library/ScriptAssemblies/Unity.Rider.Editor.dll differ
diff --git a/Library/ScriptAssemblies/Unity.TextMeshPro.Editor.dll b/Library/ScriptAssemblies/Unity.TextMeshPro.Editor.dll
index 441d9d0..4eb99ca 100644
Binary files a/Library/ScriptAssemblies/Unity.TextMeshPro.Editor.dll and b/Library/ScriptAssemblies/Unity.TextMeshPro.Editor.dll differ
diff --git a/Library/ScriptAssemblies/Unity.TextMeshPro.dll b/Library/ScriptAssemblies/Unity.TextMeshPro.dll
index a5b7476..a1651a2 100644
Binary files a/Library/ScriptAssemblies/Unity.TextMeshPro.dll and b/Library/ScriptAssemblies/Unity.TextMeshPro.dll differ
diff --git a/Library/ScriptAssemblies/Unity.Timeline.Editor.dll b/Library/ScriptAssemblies/Unity.Timeline.Editor.dll
index a93fb36..d39303e 100644
Binary files a/Library/ScriptAssemblies/Unity.Timeline.Editor.dll and b/Library/ScriptAssemblies/Unity.Timeline.Editor.dll differ
diff --git a/Library/ScriptAssemblies/Unity.Timeline.dll b/Library/ScriptAssemblies/Unity.Timeline.dll
index ada485f..6630048 100644
Binary files a/Library/ScriptAssemblies/Unity.Timeline.dll and b/Library/ScriptAssemblies/Unity.Timeline.dll differ
diff --git a/Library/ScriptAssemblies/Unity.VSCode.Editor.dll b/Library/ScriptAssemblies/Unity.VSCode.Editor.dll
index 892aa4a..9dc44a8 100644
Binary files a/Library/ScriptAssemblies/Unity.VSCode.Editor.dll and b/Library/ScriptAssemblies/Unity.VSCode.Editor.dll differ
diff --git a/Library/ScriptAssemblies/UnityEditor.TestRunner.dll b/Library/ScriptAssemblies/UnityEditor.TestRunner.dll
index 7cc74a4..4737764 100644
Binary files a/Library/ScriptAssemblies/UnityEditor.TestRunner.dll and b/Library/ScriptAssemblies/UnityEditor.TestRunner.dll differ
diff --git a/Library/ScriptAssemblies/UnityEditor.UI.dll b/Library/ScriptAssemblies/UnityEditor.UI.dll
index 7b853fa..b5b602a 100644
Binary files a/Library/ScriptAssemblies/UnityEditor.UI.dll and b/Library/ScriptAssemblies/UnityEditor.UI.dll differ
diff --git a/Library/ScriptAssemblies/UnityEngine.TestRunner.dll b/Library/ScriptAssemblies/UnityEngine.TestRunner.dll
index d004383..c4a3861 100644
Binary files a/Library/ScriptAssemblies/UnityEngine.TestRunner.dll and b/Library/ScriptAssemblies/UnityEngine.TestRunner.dll differ
diff --git a/Library/ScriptAssemblies/UnityEngine.UI.dll b/Library/ScriptAssemblies/UnityEngine.UI.dll
index 847da8f..cbb94de 100644
Binary files a/Library/ScriptAssemblies/UnityEngine.UI.dll and b/Library/ScriptAssemblies/UnityEngine.UI.dll differ
diff --git a/Library/expandedItems b/Library/expandedItems
index 2ff8cc8..400862a 100644
Binary files a/Library/expandedItems and b/Library/expandedItems differ
